src/components/labels/Label.jsx

```diff
@@ -0,0 +1,183 @@
+  gray: 'gray-primary',
+  mono: 'black',
+};
+
+export const COLORS_DEFAULT_MAP = {
+  blue: 'blue-secondary-light',
+  mint: 'mint-secondary-light',
+  lavender: 'lavender-secondary-light',
+  peach: 'peach-secondary-light',
+  mustard: 'mustard-secondary-light',
+  gray: 'gray-secondary-light',
+  mono: 'white',
+};
+
+export const LABEL_COLORS_SET = {
+  BLUE: 'blue',
+  MINT: 'mint',
+  LAVENDER: 'lavender',
+  PEACH: 'peach',
+  MUSTARD: 'mustard',
+  GRAY: 'gray',
+  MONO: 'mono',
+};
+
+type PropsType = $ReadOnly<{
+  /**
+   * Specify type of label
+   * @example <Label
+   *           type="default"
+   *           color="blue"
+   *          >
+   *            example label
+   *          </Label>
+   * @see type="default" https://styleguide.brainly.com/latest/docs/interactive.html?type="default"#labels
+   * @see type="strong" https://styleguide.brainly.com/latest/docs/interactive.html?type="strong"#labels
+   */
+  type: LabelType,
+  /**
+   * Specify color for label
+   * @example <Label
+   *           type="default"
+   *           color="blue"
+   *          >
+   *            example label
+   *          </Label>
+   * @see color="blue" https://styleguide.brainly.com/latest/docs/interactive.html?color="blue"#labels
+   * @see color="mint" https://styleguide.brainly.com/latest/docs/interactive.html?color="mint"#labels
+   * @see color="lavender" https://styleguide.brainly.com/latest/docs/interactive.html?color="lavender"#labels
+   * @see color="peach" https://styleguide.brainly.com/latest/docs/interactive.html?color="peach"#labels
+   * @see color="mustard" https://styleguide.brainly.com/latest/docs/interactive.html?color="mustard"#labels
+   * @see color="gray" https://styleguide.brainly.com/latest/docs/interactive.html?color="gray"#labels
+   * @see color="mono" https://styleguide.brainly.com/latest/docs/interactive.html?color="mono"#labels
+   */
+  color: LabelColorType,
+  /**
+   * Icons types example, see more in SG interactive
+   * @example <Label
+   *           type="default"
+   *           color="blue"
+   *           iconType="heart"
+   *          >
+   *            example label
+   *          </Label>
+   * @see type="iconType" https://styleguide.brainly.com/latest/docs/interactive.html?iconType=heart#labels
+   */
+  iconType?: IconModule.IconTypeType,
+  /**
+   * Callback, called by clicking on **close** button. If specified, button will be added automatically
+   * @example <Label
+   *           type="default"
+   *           color="blue"
+   *           iconType="heart"
+   *           onClose={() => doSomething()}
+   *          >
+   *            example label
+   *          </Label>
+   */
+  onClose?: ?(SyntheticInputEvent<HTMLDivElement>) => mixed,
+  /**
+   * Children to be rendered inside Label
+   * @example <Label
+   *           type="default"
+   *           color="blue"
+   *          >
+   *            example label
+   *          </Label>
+   */
+  children: Node,
```

Comment:
should we allow to define children? i know we did in the past, but i know not sure if it's needed here now. wdyt?